.gitignore

```diff
@@ -55,3 +55,7 @@ derby.log
 
 # Python stuff
 python/.mypy_cache/
+
+# K8s integration stuff
+integration-test/extra.yaml
+integration-test/extra.sh
```

Comment:
Oh yeah haha I see definitely this is intended to be your secrets or local specific stuff. Maybe we update the source command with another `|| true`?

Again, happy to make these changes but don't want to step on your branch without talking to you first :)

Reply:
Yeah sounds like a good call.